CHANGELOG

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.3.1] 2024-10-4
```

Comment:
The hotfix that was deployed yesterday is 1.3.1. This should be 1.3.2. Update this version in package.json as well.